doc/architecture/database_schemata.md

```diff
@@ -87,8 +87,69 @@ species }|--|| genus : ""
 genus }|--|| subfamily : ""
 subfamily }|--|| family : ""
 
+maps {
+  INT id PK
+  VARCHAR name "NOT NULL"
+  BOOLEAN is_version "NOT NULL"
+  VARCHAR version_name
+  BOOLEAN is_inactive "NOT NULL"
+  DATE last_visit
```

Comment:
more importantly, we need a date of the version

Reply:
At first I thought we could reuse creation_date, but you are right. A dedicated version_date field makes sense.

Reply:
Yes, creation of the map vs. creation of the specific version.

Reply:
The creation date doesn't change, even if the oldest versions get deleted.